### ISSUE TITLE
Add to NPM?

### ISSUE THREAD

**Issue by aethant** [Grible/timetable.js#1]
Would it be possible to add this library to NPM as well?

**Comment by stevenaanen** [comment]
Good idea. I think we could add it the quick&dirty way. The plugin is however only used for client-side code at the moment and is not compatible with the CommonJS/default way of publishing node packages. Therefore I thought bower would be more suitable.

Just out of curiosity @aethant, are you using NPM to do package management for client-side code as well?

**Comment by aethant** [comment]
@stevenaanen yes, i am, with webpack. 

Thanks!

**Comment by stevenaanen** [comment]
Okay cool, I will have a look at it. Then we should make the plugin compatible with CommonJS without breaking functionality for people that use bower or no package manager. Not sure how to do that right away.

**Comment by Marcdup1** [comment]
Is this still going to happen @stevenaanen

**Comment by stevenaanen** [comment]
@Marcdup1 unfortunately we're a bit short on time at the moment to actively work on timetable.js. If you have a pull request for this I'd be happy to add it though.

**Comment by bikegriffith** [comment]
Opened PR #52 with the changes necessary to make this happen.  I can publish for you under my npm account if you'd like, or you can do it yourself.

**Comment by bryce13950** [comment]
@bikegriffith could you publish this?

**Comment by bikegriffith** [comment]
Are there any maintainers that can give me clearance to publish, or any maintainers that are willing to publish themselves?

**Comment by bryce13950** [comment]
Unrelated but, were you able to get your version to load via an es6 style import? I tried with the current production release, and it did not seem to work.

**Comment by bikegriffith** [comment]
@bryce13950 Yes, it worked for me using both `import` and `require`.

**Comment by bryce13950** [comment]
@bikegriffith Don't mean to bother you too much on this, but do you want to publish your version? It seems like this is a bit of a dead thread.

**Comment by bikegriffith** [comment]
@bryce13950 Published as `timetable.js` (there was already a `timetable` in npm).  Please `npm install timetable.js` and let me know if it works.
